src/Illuminate/Http/Response.php

```diff
@@ -40,12 +40,12 @@ public function __construct($content = '', $status = 200, array $headers = [])
     /**
      * Set the content on the response.
      *
-     * @param  mixed  $content
+     * @param  string|null  $content
      * @return $this
      *
      * @throws \InvalidArgumentException
      */
-    public function setContent($content)
+    public function setContent(?string $content): static
```

Comment:
we can still accept mixed, and try to cast to string inside here, if not null. wider param types are allowed, and narrower return types

Reply:
Thanks, updated this.